nncf/quantization/algorithms/min_max/utils.py

```diff
@@ -68,9 +86,22 @@ def calculate_weight_quantizer_parameters(weight_tensor: np.ndarray, quantizer_c
         axes = None
     input_high = np.amax(weight_tensor, axis=axes)
     input_low = np.amin(weight_tensor, axis=axes)
-    scales = calculate_scale_level(input_high, input_low, num_bits, mode)
-    zero_points = np.zeros_like(scales, dtype=np.int64)
-    return QuantizerLayerParameters(scales.tolist(), zero_points.tolist(), mode, axis)
+    tensor_type = np.uint8 if np.all(input_low >= 0) else np.int8
+    level_low, level_high = get_level_low_level_high(tensor_type)
+    scales, zero_points = calculate_scale_level(input_high, input_low, level_low, level_high, mode)
+    return QuantizerLayerParameters(scales.tolist(), zero_points.tolist(), mode, axis, tensor_type)
+
+
+def get_level_low_level_high(tensor_type: np.dtype) -> Tuple[int, int]:
+    """
+    Returns the minimum and maximum level for the quantizer.
+
+    :param tensor_type: Value type of the tensor. Could be INT8 or UINT8.
+    :return: Minimum level and maximum level of the quantizer.
+    """
+    if tensor_type == np.uint8:
+        return 0, 255
```

Comment:
These values should be collected using the HW configuration, shouldn't they? Otherwise, these values can be used as default.

Reply:
Based on the transformation from OpenVINO - https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/quantize_linear.cpp#L61

Reply:
Ok. But what about this config? https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/hardware/configs/cpu.json#L27

Reply:
There is no possibility to use custom quantization levels for QDQ.

Reply:
I can't understand why we can't use the values from the HW config if they exist there.